fhir-core/src/main/java/com/ibm/fhir/core/FHIRVersionParam.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.core;
+
+/**
+ * Enum constants for the allowed values of the fhirVersion MIME-type parameter
+ */
+public enum FHIRVersionParam {
+    VERSION_40("4.0"),
+    VERSION_43("4.3");
+
+    private final String value;
+
+    FHIRVersionParam(String value) {
```

Comment:
javadoc

Reply:
will add and also make the constructor private

Reply:
done